packages/app/src/specs/SpecsList.vue

```diff
@@ -43,7 +43,9 @@
           t('specPage.componentSpecsHeader') : t('specPage.e2eSpecsHeader') }}
       </div>
       <div class="flex items-center justify-between">
-        <div>{{ t('specPage.lastUpdatedHeader') }}</div>
+        <div :is-git-available="isGitAvailable">
```

Comment:
Looks like we can remove this attribute from the div and the `isGitAvailable` computed value, then this file won't be touched by the PR.

Reply:
I removed that, but the localized string path was updated so this will touch the file.